.github/ISSUE_TEMPLATE/localization_suggestion.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Localization Suggestion
+about: Report a problem or suggested change to Calculator's localized content.
+title: '[Localization] '
+labels: 'Area: World-Readiness'
+assignees: ''
+---
+<!--
+Please note:  We cannot _accept_ any change to our localized resources files. These files are automatically generated from an internal localization process.
```

Comment:
Please manually wrap this at 100 chars

Reply:
We do not want to manually wrap these in comments for tempaltes. It casues this: 
![image](https://user-images.githubusercontent.com/1414423/55652865-9e60b580-57a1-11e9-98bf-492893df0d2e.png)